docs/DeveloperGuide.md

```diff
@@ -346,74 +346,269 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
+`* *` | user | sync my task list to Google or Microsoft | view my tasks through their respective calendars
+`* *` | user | retrieve previously typed commands using the UP and DOWN keys and execute them directly | save time having to retype similar commands repeatedly
+`* *` | user | postpone an existing task via a specific command | save time having to specifically enter the new deadline
+`* *` | user | attach related documents and links to my tasks | conveniently access documents needed for me to work on the tasks
+`* *` | new user | view command hints when typing commands | ensure my commands are correct
+`* *` | user | block off multiple slots for the same task and release the unused slots when the exact timing of a task is confirmed | avoid having to add multiple copies of the same task to multiple potential time slots
+`* *` | user | view the slots at which I have not planned for any task to be completed | find a suitable slot for new tasks easily
+`*` | user sharing this computer with other users | switch between different accounts on this task manager | share this application with the other users of this computer
+`*` | user | backup my entire task list to the cloud or external storage via export | have extra redundancy against system failures
+`*` | user | create shared tasks across different users | track the progress of other users for shared tasks
+`*` | user | import previously exported data back | continue from where I left off
+`*` | user | receive a daily email at a preferred time that contains tasks due that day | plan my schedule for that day effectively
+
 
 {More to be added}
 
 ## Appendix B : Use Cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TaskManager` and the **Actor** is the `user`, unless specified otherwise)
+
+#### Use case: Show possible commands to be executed
+
+**MSS**
+
+1. User types a letter or a sequence of letters
+2. System shows a list of commands that contain that letter or sequence of letters
+3. User selects specific command in list
+4. System autocompletes user's command
+Use case ends. 
+
+**Extensions**
+
+2a. No command contains letter or sequence of letters typed by user
+
+> Use case ends
+
+2b. User makes changes to input
+
+> Use case resumes at step 2
+
+#### Use case: Change save location
+
+**MSS**
+
+1. User clicks on "Change Save Location"
+2. System displays a directory chooser
+3. User selects a directory
+4. System changed the save location to the one selected
+5. System displays a message 
+Use case ends
+
+**Extensions**
+
+1a. User pressed "Alt + S" keys
+
+> Use case resumes at step 2
+
+3a. User clicks on "Cancel"
 
-#### Use case: Delete person
+> Use case ends
+
+#### Use case: Switch to console with hotkeys
+
+**MSS**
+
+1. User presses on predefined hotkey
+2. System displays main window
```

Comment:
this use case seems incomplete. 
may be you need to specify what hot keys do what and then write steps for at least a few of them.